garb/garb_recv_loop.cpp

```diff
@@ -227,21 +253,22 @@ RecvLoop::one_loop()
                     // Note that in case of termination, both sst_terminated_ and
                     // sst_ended_ flags are set. We need to test sst_terminated_ first.
                     if (sst_terminated_) {
-                        log_info << "SST script already terminated";
-                        rcode_ = process_->wait();
+                        // Donor exited in the middle of transfer
+                        // or garbd had to terminate sst script after the transfer (sst script didn't self-exited)
+                        log_info << "SST script terminated has been terminated";
```

Comment:
i don't understand what this log means

Reply:
This means that my English is terribly bad...
Fixed.